include/llvm/MC/MCWinCOFFStreamer.h

```diff
@@ -53,6 +53,7 @@ class MCWinCOFFStreamer : public MCObjectStreamer {
   void EmitCOFFSafeSEH(MCSymbol const *Symbol) override;
   void EmitCOFFSectionIndex(MCSymbol const *Symbol) override;
```

Comment:
Do we ever need EmitCOFFSectionIndexValue also that accepts MCExpr instead of MCSymbol?